src/Core/Utilities/JsonHelpers.cs

```diff
@@ -129,4 +130,63 @@ public override void Write(Utf8JsonWriter writer, DateTime? value, JsonSerialize
             writer.WriteStringValue(CoreHelpers.ToEpocMilliseconds(value.Value).ToString());
         }
     }
+
+    /// <summary>
+    /// Allows reading a string from a number or string, should only be used on string properties
+    /// </summary>
+    public class PermissiveStringConverter : JsonConverter<string>
+    {
+        internal static PermissiveStringConverter Instance = new();
+
+        public override string Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            return reader.TokenType switch
+            {
+                JsonTokenType.String => reader.GetString(),
+                JsonTokenType.Number => reader.GetInt64().ToString(),
+                JsonTokenType.Null => null,
+                _ => throw new JsonException($"Unsupported TokenType: {reader.TokenType}"),
+            };
+        }
+
+        public override void Write(Utf8JsonWriter writer, string value, JsonSerializerOptions options)
+        {
+            writer.WriteStringValue(value);
+        }
+    }
+
+    /// <summary>
+    /// Allows reading a string from a number or string, should only be used on string properties
+    /// </summary>
+    public class PermissiveStringEnumerableConverter : JsonConverter<IEnumerable<string>>
+    {
+        public override IEnumerable<string> Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+        {
+            if (reader.TokenType != JsonTokenType.StartArray)
+            {
+                throw new JsonException("Should only be used on a JSON Array");
+            }
+
+            var stringList = new List<string>();
+
+            while (reader.Read() && reader.TokenType != JsonTokenType.EndArray)
+            {
+                stringList.Add(PermissiveStringConverter.Instance.Read(ref reader, typeof(string), options));
+            }
+
+            return stringList;
+        }
+
+        public override void Write(Utf8JsonWriter writer, IEnumerable<string> value, JsonSerializerOptions options)
+        {
+            writer.WriteStartArray();
```

Comment:
What happens if `value` is `null`? You should be calling `WriteNullValue()` instead of `StartArray()`.

Reply:
Similar to reading a null and returning an empty Enumerable should I just write the start array and end array (`[]`)?

Reply:
Yeah, I think that's fine for this scenario, since we're assuming the .NET type is of `IEnumerable<string>` and therefore _shouldn't_ be `null`... I've seen it go to JSON both ways, `[]` produces smaller JSON than a literal `null`, so I'm good with it.

Reply:
After making tests for this. When the value is null it doesn't even send it to my converter and writes null for me so I can't do this.